jetpack.php

```diff
@@ -154,11 +167,14 @@ function jetpack_admin_missing_autoloader() { ?>
 	return;
 }
 
-// Load all the packages.
+/**
+ * Load all the packages.
+ *
+ * We want to fail gracefully if `composer install` has not been executed yet, so we are including the autoloader.
+ * If `include` isn't able to load the autoloader, let's log the failure, pause Jetpack, and display a nice admin notice.
+ */
 $jetpack_autoloader = JETPACK__PLUGIN_DIR . '/vendor/autoload.php';
-if ( is_readable( $jetpack_autoloader ) ) {
```

Comment:
At a first glance, the previous code seemed a bit more clear about what the intentions are. Could you elaborate on the benefit of the new approach over the previous one?

Reply:
This was to address a concern @lezama raised that the old code ran `is_readable`, which is a disk read to check if the file exists and has permissions, then `require` the file, which is a second disk read.

This does the same thing with a single disk read.

Reply:
To be fair, I prefer 2 disk reads here (speed differences should be minimal), instead of suppressing `include`'s warning.

Reply:
Restored original in 0238eab32222d21deda286d55b3551c2af48ee7b ( cc: @lezama )